### ISSUE TITLE
Listings don't show default preview image

### ISSUE THREAD

**Issue by richardsprague** [quarto-dev/quarto-cli#7102]
### Bug description

One of the many .md files in my `downloads` directory contains as its first image the following markdown, which displays correctly upon rendering.

```md

![Almond Flour Bread](https://onedrive.live.com/embed?resid=8BC6084B92FFA451%21543979&authkey=%21AC9ByIteI8i__iA&width=660&height=999999?no.jpg)
```

However, that image doesn't show on the listing page set up like this: 

```yml
---
title: All Recipes
listing: 
    categories: true
    contents: "downloads/"
    type: grid
    feed: true
show_toc: TRUE
---
```

instead it generates an empty placeholder card with this html

```html
<div class="listing-item-img-placeholder card-img-top" style="height: 150px;">&nbsp;</div>
```

quarto --version
1.4.395

**Comment by richardsprague** [comment]
FYI, v1.3.450 correctly generates this html:

```html
<p class="card-img-top"><img src="https://onedrive.live.com/embed?resid=8BC6084B92FFA451!543979&amp;authkey=!AC9ByIteI8i__iA&amp;width=660&amp;height=999999?no.jpg" style="height: 150px;" class="thumbnail-image card-img"></p>
```

**Comment by dragonstyle** [comment]
This is currently by design. We've run into a number of cases where we automatically detect an undesirable image as the preview image for a listing item, so we narrowed it to filter out remote urls when auto detecting.

(here is one example https://github.com/quarto-dev/quarto-cli/issues/6091)

We decided to narrow the automatic detection because while it is possible to specify an explicit image using the `image` metadata, it isn't possible to prevent the selection of an image with metadata. So adding the following to your document resolves the issue:

```yaml
image: https://onedrive.live.com/embed?resid=8BC6084B92FFA451%21543979&authkey=%21AC9ByIteI8i__iA&width=660&height=999999?no.jpg
```
